src/components/Footer.tsx

```diff
@@ -213,20 +201,78 @@ const Logo = styled.div`
 const FooterLinkGroups = styled.div`
 	display: flex;
 	flex: 1;
+	width: 100%;
+
+	@media(max-width:768px){
+		flex-wrap: wrap;
+	}
 `;
 
 const FooterLinkGroup = styled.div`
-	flex: 1;
+	// flex: 1;
 	display: flex;
 	flex-direction: column;
-	gap: 2rem;
+	gap: 3rem;
+	width: 50%;
+
 	> h5 {
 		margin: 0;
 		font-size: 1.2rem;
 	}
+	@media(max-width:768px){
+		width: 50%;
```

Comment:
overridng the exact same css attribute and value is redundant. Review this please

Reply:
Done.